addons/web/static/tests/fields/field_utils_tests.js

```diff
@@ -2,6 +2,7 @@ odoo.define('web.field_utils_tests', function (require) {
 "use strict";
 
 var core = require('web.core');
+var ajax = require('web.ajax');
```

Comment:
unused (hopefully)

Reply:
Yup, past test was downloading the moment locales, nim made me change that